internal/gps/lock.go

```diff
@@ -195,6 +196,15 @@ func (lp LockedProject) Packages() []string {
 	return lp.pkgs
 }
 
+func (lp LockedProject) String() string {
```

Comment:
Now that this has been extracted into a function, would you mind adding a small unit test for it?